core-plugins/src/main/java/co/cask/hydrator/plugin/transform/LogParserTransform.java

```diff
@@ -198,6 +205,17 @@ private String getLog(StructuredRecord input) {
     }
   }
 
+  private boolean validateInputSchemaType(Schema.Type inputSchemaType) {
```

Comment:
this should take a Schema, not a Schema.Type.  We expect all Schemas to be of type Record.